diff --git a/source/elektron/md/mdLibTest/CMakeLists.txt b/source/elektron/md/mdLibTest/CMakeLists.txt
index ef2797e4..203750d0 100644
--- a/source/elektron/md/mdLibTest/CMakeLists.txt
+++ b/source/elektron/md/mdLibTest/CMakeLists.txt
@@ -3,6 +3,10 @@ cmake_minimum_required(VERSION 3.15)
 add_executable(mdLibTest mdRuntimeTest.cpp)
 target_link_libraries(mdLibTest PRIVATE mdLib)
 
+add_executable(mdTraceTool mdTraceTool.cpp)
+target_link_libraries(mdTraceTool PRIVATE mdLib)
+set_property(TARGET mdTraceTool PROPERTY FOLDER "Elektron/test")
+
 add_executable(mmBootFirmwareTest mmBootFirmwareTest.cpp)
 target_link_libraries(mmBootFirmwareTest PRIVATE mdLib)
 add_test(NAME mmBootFirmwareTest COMMAND mmBootFirmwareTest)
